### PR TITLE
Coredev production

### PR DESCRIPTION
**Pull request by hreinking**
rebasing this branch is getting harder and harder, so I'm merging the dev site and start doing the same approach as with the other production branches

### CODE REVIEW

File: hieradata/site/dev.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+# resolv.conf allows for a maximum of 6 search domains with a max length of 256
+# characters.
+# @see man 5 resolv.conf
+dhcp::dnsdomain: &dnsdomains
+  - "ls.lsst.org"
+  - "lsst.org"
+# resolv.conf allows for a maximum of 3 nameservers
+# @see man 5 resolv.conf
+dhcp::nameservers: &nameservers
+  - "139.229.134.7"  # dns1.ls.lsst.org
```
Comment (adrienthebo) [inline hieradata/site/dev.yaml:68]:
```suggestion
  - "139.229.134.7"  # dns1.dev.lsst.org
```

File: hieradata/org/lsst/role/dev_hyp.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
```
Comment (adrienthebo) [inline hieradata/org/lsst/role/dev_hyp.yaml:1]: Is there a difference between `dev_hyp` and `hypervisor`?

Comment (hreinking) [reply hieradata/org/lsst/role/dev_hyp.yaml:1]: dev_hyp is for the server that is acting as core2

File: hieradata/site/dev.yaml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+# - dhcp::nameservers
+# - dhcp::ntpservers
+#
+# @see site/ls/role/foreman.yaml
+
+# resolv.conf allows for a maximum of 6 search domains with a max length of 256
+# characters.
+# @see man 5 resolv.conf
+dhcp::dnsdomain: &dnsdomains
+  - "ls.lsst.org"
```
Comment (adrienthebo) [inline hieradata/site/dev.yaml:63]: Should this list include `dev.lsst.org`?

File: hieradata/site/dev/role/graylog.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+---
+classes:
+  - "profile::core::common"
+  - "role_graylog"
```
Comment (adrienthebo) [inline hieradata/site/dev/role/graylog.yaml:4]: Are we still deploying graylog via Puppet in the dev environment?

File: hieradata/org/lsst/role/dev_hyp.yaml (hunk 2 above)
Comment (adrienthebo) [reply hieradata/org/lsst/role/dev_hyp.yaml:1]: We have a `core2.dev.lsst.org`? And how does this vary from the `hypervisor` role?

File: hieradata/site/dev.yaml [hunk 5]
```diff
@@ -0,0 +1,92 @@
+---
+easy_ipa::ipa_master_fqdn: "ipa1.dev.lsst.org"
+#
+rsyslog::config::actions:
+#Send copy to logs to GKE Graylog instance
```
Comment (adrienthebo) [inline hieradata/site/dev.yaml:5]: Is this still in GKE or on Ruka?

File: hieradata/site/dev.yaml (hunk 3 above)
Comment (hreinking) [reply hieradata/site/dev.yaml:64]: Adding it! thnks

File: hieradata/site/dev.yaml (hunk 5 above)
Comment (hreinking) [reply hieradata/site/dev.yaml]: Fixed it! thnks

File: hieradata/site/dev/role/graylog.yaml (hunk 4 above)
Comment (hreinking) [reply hieradata/site/dev/role/graylog.yaml]: deleted

**Pull request merged**